README.md

```diff
@@ -51,15 +39,27 @@ BLT provides an automation layer for testing, building, and launching Drupal 8 a
     ```
     $ composer install
     ```
-6. Route the site of your choice in `sites.local.php`.
-   ```
-   $sites['8888.localhost'] = 'mysite';
-   ```
-7. Configure `local.settings.php` to override Lando defaults.
-8. Start the built-in PHP server.
+6. Configure `local.settings.php` to connect to the MySQL instance:
+    ```
+    'database' => $db_name,
+    'username' => 'root',
+    'password' => '',
+    'host' => 'localhost',
+    'port' => '3306',
+    ```
+6. If the database does not exist, use `drush sql:create` to create it.
+    ```
+    drush -l mysite sql:create
+    ```
```

Comment:
I assume this is missing: `blt drupal:sync:all-sites` after the databases have been created